source/sink-connector/configuration-properties/write-strategies.txt

```diff
@@ -32,6 +41,13 @@ configuration:
 Strategies
 ----------
 
+.. tip::
+   
+   The example configurations in the following table show how to set only
+   the ``writemodel.strategy`` property. To select a write model
+   strategy as your delete write model, use the same class
+   names as values of the ``delete.writemodel.strategy`` property.
```

Comment:
Issue:
I think the description is slightly verbose and therefore difficult to understand how to specify the delete writemodel strategy.

Suggestion:
I think as this page has gotten more detailed, the distance between "Use the strategies in the following table" and the Strategies table has gotten too far apart, which makes this callout slightly more difficult to understand.

I think reorganizing the information by making all the following changes would be helpful:

- In the Overview, create two subheadings for the types of strategies. Or, alternatively, get rid of the Overview and create top level headings for describing the Write Model and Delete Write Model settings. 
- Introduce each setting separately, explain what they're used for, and mention that they can be set to any of the values listed in the Strategies section. I think describing how the connector deals with tombstone events is important to identify as a separate item because it is not optional (there's a default value for delete.writemodel.strategy).
- Since the Overview shows how to set each of them, change the Strategies table entries to only show the class name.